packages/web-app-files/tests/unit/components/SideBar/Shares/Collaborators/InviteCollaborator/__snapshots__/ExpirationDatepicker.spec.ts.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InviteCollaborator ExpirationDatepicker only gets displayed if share expiration is supported 1`] = `<!--v-if-->`;
```

Comment:
This wasn't a snapshot test before and just checked for an empty string, we made it a snapshot test to make adoptions for serializer changes easier.